File: messageGo/grpc/pubsubEventData/service_started.go
package pubsubEventData

import (
	base_data "game-message-core/grpc/baseData"
)

type ServiceStartedEvent struct {
	MsgVersion int64                 `json:"msgVersion"` // 消息版本号 值为毫秒时间戳
	Service    base_data.ServiceData `json:"service"`    //
}

File: messageDotNet/grpc/EventData/ServiceStarted.cs
using System;
using UnityEngine;

/// <summary>
/// service 销毁事件
/// </summary>
[Serializable]
public class ServiceStartedEvent
{
    // 消息版本号 值为毫秒时间戳
    public long MsgVersion;
    public ServiceData Service;

    public string ToJson()
    {
        return JsonUtility.ToJson(this);
    }
}